Common/UnitDefinitions/PorousMediumPermeability.json

```diff
@@ -0,0 +1,54 @@
+﻿{
+  "Name": "PorousMediumPermeability",
+  "BaseUnit": "SquareMeter",
+  "XmlDoc": "In fluid mechanics, permeability is the measure of the ability of a porous material to allow fluids to pass through it.",
+  "XmlDocRemarks": "https://en.wikipedia.org/wiki/Permeability_(Earth_sciences)",
+  "BaseDimensions": {
+    "L": 2
+  },
+  "Units": [
+    {
+      "SingularName": "Darcy",
+      "PluralName": "Darcys",
+      "FromUnitToBaseFunc": "{x} * 9.869233e-13",
+      "FromBaseToUnitFunc": "{x} / 9.869233e-13",
+      "Prefixes": [ "Micro", "Milli" ],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "D" ]
```

Comment:
Interesting: https://en.wikipedia.org/wiki/Darcy_(unit)

> The unit abbreviation "d" is not capitalized (contrary to industry use).

I guess we should go with industry use.